doc/source/whatsnew/v0.25.0.rst

```diff
@@ -265,6 +265,8 @@ Optional libraries below the lowest tested version may still work, but are not c
 +-----------------+-----------------+
 | pyarrow         | 0.9.0           |
 +-----------------+-----------------+
+| pyreadstat      | 0.2.5           |
```

Comment:
this is not needed here; this is only for changes

Reply:
Done.